Fix `npm` installation in `Dockerfile-dev`

Out of a sudden, `RUN npm install -g npm@latest` yields the following error when building `Dockerfile-dev`:

```text
#8 0.282 /bin/sh: 1: npm: not found
```

This should be fixed. This is also a great occasion to migrate from `debian:bookworm-slim` to `ubuntu:latest` as requested in the discussion of #109.

Full log:

```text
Building stemmaweb
[+] Building 0.9s (9/14)
=> [internal] load build definition from Dockerfile-dev 0.0s
=> => transferring dockerfile: 1.14kB 0.0s
=> [internal] load .dockerignore 0.0s
=> => transferring context: 34B 0.0s
=> [internal] load metadata for docker.io/library/debian:bookw 0.4s
=> [ 1/10] FROM docker.io/library/debian:bookworm-slim@sha256: 0.0s
=> [internal] load build context 0.0s
=> => transferring context: 30.32kB 0.0s
=> CACHED [ 2/10] RUN apt-get update && apt-get install -y 0.0s
=> CACHED [ 3/10] RUN update-alternatives --install /usr/bin/p 0.0s
=> CACHED [ 4/10] RUN apt-get update && apt-get install -y 0.0s
=> ERROR [ 5/10] RUN npm install -g npm@latest 0.3s
------
> [ 5/10] RUN npm install -g npm@latest:
#8 0.282 /bin/sh: 1: npm: not found
------
executor failed running [/bin/sh -c npm install -g npm@latest]: exit code: 127
ERROR: Service 'stemmaweb' failed to build : Build failed
make: *** [build-dev] Error 1
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -65,7 +65,7 @@ jobs:
     needs: set_vars
     name: End-to-end Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 60
     steps:
       - name: Clone repository
         uses: actions/checkout@v3

--- a/Dockerfile-dev
+++ b/Dockerfile-dev
@@ -1,4 +1,4 @@
-FROM debian:bookworm-slim
+FROM ubuntu:latest
 
 # Install Python and aux packages
 RUN apt-get update && apt-get install -y \
@@ -22,7 +22,8 @@ RUN apt-get update && apt-get install -y \
     lsb-release &&  \
     curl -fsSL https://deb.nodesource.com/setup_18.x | bash - &&  \
     apt-get install -y nodejs
-# Install npm
+
+# Update npm (got installed in the previous layer with `nodejs`)
 RUN npm install -g npm@latest
 
 # Install utilities

